website/docs/validating-admission-policy.md

```diff
@@ -40,6 +40,10 @@ In summary, these are potential options when running Gatekeeper:
 
 Find out more about different [enforcement points](enforcement-points.md)
 
+:::note
+CEL validation in constraint templates through Gatekeeper is in beta and `--experimental-enable-k8s-native-validation` is turned on by default. Set --experimental-enable-k8s-native-validation=false` to disable evaluating CEL in constraint templates.
```

Comment:
https://github.com/open-policy-agent/gatekeeper/pull/3476/files#diff-6f51397980ac58fff52f433210aa369e522c495bdd049d01ae538eaa3b9118b1R6-R8 should be updated too for 3.17 and this text can be added there instead of here.

> `Feature State`: Gatekeeper version v3.17 (beta)

Reply:
`ValidatingAdmissionPolicy` is different than `CEL validation in Gatekeeper`, wouldn't updating in the section you mentioned be missleading? It might seem like we moved VAP/VAPB generation to `beta`. We probably need to distinguish between generation and CEL validation. How about the below?

> VAP management through Gatekeeper: 
  Feature State: Gatekeeper version v3.16 (alpha)
❗ This feature is alpha, subject to change (feedback is welcome!). It is enabled by default.

> CEL validation in Gatekeeper: 
  Feature State: Gatekeeper version v3.17 (beta)
❗ This feature is beta, subject to change (feedback is welcome!). It is enabled by default. Set --experimental-enable-k8s-native-validation=false` to disable evaluating CEL in constraint templates.

Reply:
CEL validation in Gatekeeper:
Feature State: Gatekeeper version v3.17 (beta)
❗ This feature is beta, subject to change (feedback is welcome!). It is enabled by default. Set --experimental-enable-k8s-native-validation=false` to disable evaluating CEL in constraint templates.

VAP management through Gatekeeper:
Feature State: Gatekeeper version v3.16 (alpha)
❗ This feature is alpha, subject to change (feedback is welcome!). It is disabled by default unless explicitly enabled via feature flag and/or via constraint template.